tools/interu/src/cli.rs

```diff
@@ -0,0 +1,39 @@
+use std::path::PathBuf;
+
+use argh::FromArgs;
+
+#[derive(Debug, FromArgs)]
+/// Expand configuration into key-value pairs used by the run-integration-test action
```

Comment:
FYI, we use `key=value` below, in case you want to make them consistent.

Reply:
Done in 21089ed